worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingReportUpdateWorker.java

```diff
@@ -137,38 +138,81 @@ private JobTrackingWorkerResult processTrackingEvent() throws InterruptedExcepti
             throws JobReportingException
     {
         final List<JobTrackingWorkerDependency> jobDependencyList = new ArrayList<>();
+        List<String> jobTaskIds = new ArrayList<>();
+
+        String partitionId="";
+        String jobId="";
```

Comment:
We should keep the groups rather than dispatching after each change as discussed.